torchtune/_cli/hf_upload/LICENSE

```diff
@@ -1,125 +0,0 @@
-LLAMA 2 COMMUNITY LICENSE AGREEMENT
```

Comment:
so this was here on purpose but it's an artifact of when we only supported llama2